### PR TITLE
Fixing runtime properties in K8 containers

### PR DESCRIPTION
**Pull request by lmanohara**

### CODE REVIEW

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 1]
```diff
@@ -355,14 +355,20 @@ var buildDeploymentConfigFromDB = function(applicationName, applicationRevision)
...
     var appBaseImageName = tenantDockerRegistryUrl + "/" + applicationName;
     container1.setContainerName(applicationName);
     container1.setBaseImageName(appBaseImageName);
     container1.setBaseImageVersion(applicationRevision);
+    //setting runtime properties to the container
     var envs1 = new java.util.HashMap();
+    var runtimePropertyIterator = runtimeProperties.iterator();
+    while(runtimePropertyIterator.hasNext()){
+        var runtimeProperty = runtimePropertyIterator.next();
+        envs1.put(runtimeProperty.getPropertyName(), runtimeProperty.getPropertyValue());
```
Comment (punnadi) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: Rename 'envs1'

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 2]
```diff
@@ -175,8 +175,8 @@ var addApplication = function (applicationName, applicationRevision, application
     //============================================================================================================
 
     //==================================== Kubernetes deploy block ===============================================
-
-    var containers = getContainers(tenantDockerRegistryUrl,applicationName,applicationName, applicationRevision);
+    var runtimeProperties = application.getRuntimeProperties();
+    var containers = getContainers(tenantDockerRegistryUrl,applicationName,applicationName, applicationRevision,runtimeProperties);
```
Comment (kasundsilva) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: Please use the correct formatting

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag (hunk 1 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: This is fixed in master code
https://github.com/wso2/app-cloud/blob/master/modules/jaggeryapps/appmgt/src/modules/application/application.jag#L513

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag (hunk 2 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: This is fixed by pull request 
https://github.com/wso2/app-cloud/compare/master...lmanohara:master#diff-4fa2fd04c7be70a4d733dbf56fc77947R259

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 3]
```diff
@@ -175,8 +175,8 @@ var addApplication = function (applicationName, applicationRevision, application
     //============================================================================================================
 
     //==================================== Kubernetes deploy block ===============================================
-
-    var containers = getContainers(tenantDockerRegistryUrl,applicationName,applicationName, applicationRevision);
+    var runtimeProperties = application.getRuntimeProperties();
```
Comment (prabathabey) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: Fix formatting issues in this particular line.

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 4]
```diff
@@ -355,14 +355,20 @@ var buildDeploymentConfigFromDB = function(applicationName, applicationRevision)
...
     var containerList = new java.util.ArrayList();
     var container1 = new Packages.org.wso2.appcloud.provisioning.runtime.beans.Container();
     var appBaseImageName = tenantDockerRegistryUrl + "/" + applicationName;
     container1.setContainerName(applicationName);
     container1.setBaseImageName(appBaseImageName);
     container1.setBaseImageVersion(applicationRevision);
+    //setting runtime properties to the container
     var envs1 = new java.util.HashMap();
+    var runtimePropertyIterator = runtimeProperties.iterator();
+    while(runtimePropertyIterator.hasNext()){
```
Comment (prabathabey) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: Fix formatting issues in this particular line.

Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: Yes this has a formatting issues, but this line has been removed from the latest code.

**Pull request merged**